addons/project/static/src/views/project_task_list/project_task_list_controller.js

```diff
@@ -0,0 +1,40 @@
+/** @odoo-module */
+
+import { ListController } from "@web/views/list/list_controller";
+import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+import { sprintf } from "@web/core/utils/strings";
+import { session } from "@web/session";
+
+export class ProjectTaskListController extends ListController {
+    async onDeleteSelectedRecords() {
+        const root = this.model.root;
```

Comment:
Shouldn't we check something here?

Reply:
> Shouldn't we check something here?

What do you mean?

Reply:
Do we display the confirmation dialog even if none of the selected task has any subtask?